wikimedia/wikipedia-ios
# Native Editor - Input view design + icon work

**Phabricator:** 
https://phabricator.wikimedia.org/T344508

### Notes
This wraps up the task with finalizing the new input view design as requested in [this comment](https://phabricator.wikimedia.org/T344508#9400570). I also did another pass at icons in the toolbars, find and replace, and navigation bar to improve dynamic type support.

### Test Steps
1. Open an article editor on Staging. Do light regression testing on buttons.
2. Tap the new text format icon to show the new input view. Do light regression testing on buttons.
3. Tap the reading themes navigation button. Do light regression testing on controls.
4. Confirm theming works on buttons and new input view (note there may be further tweaks on colors once this goes through design review).
5.  Change dynamic type. Confirm toolbar buttons, navigation bar buttons and new input view buttons expand or shrink as needed.
